### PR TITLE
Adds operator-version to k8s resources

### PR DESCRIPTION
**Pull request by tchellomello**
cc: @rooftopcellist 

After working on a few issues, it is a good troubleshooting resource to be able to determine which `awx-operator` version was used to create a given resource (`deployment`, `statefulset`, `secret`,  `configmap`, `awx`, etc..). 

This PR introduces this ability and added the `operator-version` as  k8s label on the expected resources. For example:

```yaml
# kubectl describe awx awx-toca  
Name:         awx-toca
Namespace:    default
Labels:       app.kubernetes.io/component=awx
              app.kubernetes.io/managed-by=awx-operator
              app.kubernetes.io/name=awx-toca
              app.kubernetes.io/operator-version=0.10.0
              app.kubernetes.io/part-of=awx-toca
Annotations:  <none>
API Version:  awx.ansible.com/v1beta1
Kind:         AWX

# kubectl get all -l  "app.kubernetes.io/operator-version=0.10.0"
NAME                            TYPE        CLUSTER-IP     EXTERNAL-IP   PORT(S)    AGE
service/awx-devel-ee-postgres   ClusterIP   None           <none>        5432/TCP   35h
service/awx-devel-ee-service    ClusterIP   10.233.45.62   <none>        80/TCP     35h
service/awx-toca-service        ClusterIP   10.233.9.69    <none>        80/TCP     45h

NAME                           READY   UP-TO-DATE   AVAILABLE   AGE
deployment.apps/awx-devel-ee   1/1     1            1           35h
deployment.apps/awx-toca       1/1     1            1           45h

NAME                                     READY   AGE
statefulset.apps/awx-devel-ee-postgres   1/1     35h

```

This will provide a quick mechanism for cross-checking information and debug.

### CODE REVIEW

File: deploy/awx-operator.yaml
Comment (tchellomello) [inline deploy/awx-operator.yaml:639]: IHMO should be `value: devel`. Then we should bump this to `expected_bumped_version`  during the release process.
Since we don't have that in place yet, I made it this `0.9.0` for consistency, I would be happy to add the logic so we can have it well placed.

**Review by rooftopcellist — APPROVED** [review]
LGTM, thanks for adding these labels, this will make debugging later easier.

**Comment by tchellomello** [comment]
Waiting for another approval and then I'll merge then! Thanks!

**Review by shanemcd — APPROVED** [review]

**Pull request merged**
